milestones/3_human_brain_mapping_journal_submission/experiments/ex_2_pli_generation_optimization/example_pli_analysis.m

```diff
@@ -0,0 +1,214 @@
+%% Yacine Mahdid 17 Septembre 2020
+
+function example_pli_analysis(p_id, epoch)
+    %% EXAMPLE PLI ANALYSIS this is an example script to run the pli generation as fast as possible
+    % input:
+    % p_id: is the id of the participant (e.g. MDFAXX)
+    % epoch: is the epoch of interest (e.g. eyesclosed_1)
+
+    %% Path Setup
+    % Local Source
+    %{
+    INPUT_DIR = "/media/yacine/My Book/datasets/consciousness/AEC vs wPLI/source localized data/";
+    OUTPUT_DIR = "/media/yacine/My Book/test_result/ex_1_pli_generation_comparison/pli_surrogate_graphs/";
+    %}
+
+    % Remote Source
+    %
+
+    % Create a "local" cluster object
+    distcomp.feature( 'LocalUseMpiexec', false ); % This was because of some bug happening in the cluster
+    local_cluster = parcluster('local');
+
+    % Modify the JobStorageLocation to $SLURM_TMPDIR
+    local_cluster.JobStorageLocation = strcat('/scratch/yacine08/', getenv('SLURM_JOB_ID'));
+
+    % Start the parallel pool
+    parpool(local_cluster, str2num(getenv('SLURM_CPUS_ON_NODE')))
+
+    INPUT_DIR = "/lustre03/project/6010672/yacine08/aec_vs_pli/data/source_localized_data/";
+    OUTPUT_DIR = "/lustre03/project/6010672/yacine08/aec_vs_pli/result/ex_2_pli_generation_optimization/";
+    NEUROALGO_PATH = "/lustre03/project/6010672/yacine08/NeuroAlgo";
+
+    % Add NA library to our path so that we can use it
+    addpath(genpath(NEUROALGO_PATH));
+
+    %}
+
+    % indice of the scalp regions
```

Comment:
Can you specify here what is meant by scalp regions? Are these electrodes or regions of interest in the source space atlas? If it's electrodes, can you specify if you are keeping 97 or 99 scalp electrodes?

Reply:
I don't see where these brackets opened.